Podfile

```diff
@@ -76,7 +76,7 @@ end
 
 def shared_test_pods
     pod 'OHHTTPStubs/Swift', '~> 9.1.0'
-    pod 'OCMock', '3.4.3'
+    pod 'OCMock', '~> 3.4.3'
```

Comment:
At this time, the latest OCMock is [v3.8.1](https://github.com/erikdoe/ocmock/releases/tag/v3.8.1).

Is this change merely a way to get potential bug fixes in version `3.4.x`? Should we block out some time to upgrade the library to `~> 3.8`?